core/src/main/java/org/infinispan/eviction/impl/PassivationManagerImpl.java

```diff
@@ -115,7 +115,9 @@ public void passivateAll() throws PersistenceException {
          return CompletableFutures.completedNull();
 
       long start = timeService.time();
-      CONTAINER.passivatingAllEntries();
+      if (CONTAINER.isDebugEnabled()) {
+         CONTAINER.debug("Passivating all entries to disk");
```

Comment:
Just curious what would it take to i18n this?

Reply:
I guess we could have kept it in the `Log` interface, only with level `DEBUG`... I see we have plenty of those already.

Reply:
Yes, that is a relic from the past that we must fix: debug messages should not be up for i18n

Reply:
Right! Debug and trace messages should not be I18N.